src/rt/cover.d

```diff
@@ -149,10 +149,80 @@ uint parseNum(const(char)[] s)
 T min(T)(T a, T b) { return a < b ? a : b; }
 T max(T)(T a, T b) { return b < a ? a : b; }
 
+version (unittest) __gshared bool inUnittest;
+
+string parseMergeOpt(string opt) @nogc nothrow
```

Comment:
Care to add some documentation? :-)

Reply:
Anywhere besides the changelog where this can be documented?

Reply:
`private` also

Reply:
Ok, there is camp that says that `private` functions doesn't need a documentation, but I wouldn't mind just a single line comment above this function to says what it is doing.

Reply:
Ah, OK.